todolist/app/screens/App/index.js

```diff
@@ -1,21 +1,26 @@
-import React from 'react';
-import { View, SafeAreaView } from 'react-native';
+import { createStackNavigator, createAppContainer, TabNavigator, createBottomTabNavigator } from 'react-navigation';
 
-import styles from './styles';
-import Title from '../../components/Title';
-import Footer from '../../components/Footer';
-import Input from '../../components/Input';
-import List from '../../components/List';
+import Todo from '../Todo'
+import ListBook from '../ListBook';
 
-function App() {
-  return (
-    <SafeAreaView style={styles.container}>
-      <Title />
-      <Input />
-      <List />
-      <Footer />
-    </SafeAreaView>
-  );
-};
+const Tabs = createBottomTabNavigator({
```

Comment:
all of this, should be in a file called 'screens.js' inside app's folder (just a file, not folder)